docs/sources/shared/datasources/tempo-search-traceql.md

```diff
@@ -0,0 +1,103 @@
+---
+headless: true
+---
+
+<!--
+This file documents the Search query type for the Tempo data source. It is available as a public preview.
+
+This shared file is included in these locations:
+/grafana/docs/sources/datasources/tempo/query-editor/index.md
+
+If you make changes to this file, verify that the meaning and content are not changed in any place where the file is included.
+
+Any links should be fully qualified and not relative: /docs/grafana/
+ -->
```

Comment:
Nice! Really helpful comment.

Another place where we can consider using "Markdown comments" instead of HTML comments to avoid this ending up the page HTML.

Reply:
Thank you. I've updated the comments to use the Markdown comments you highlighted earlier.